Userland/Libraries/LibWeb/SVG/SVGElement.idl

```diff
@@ -1,7 +1,9 @@
 #import <CSS/ElementCSSInlineStyle.idl>
 #import <DOM/Element.idl>
-#import <HTML/HTMLElement.idl>
+#import <DOM/EventHandler.idl>
 #import <HTML/DOMStringMap.idl>
+#import <HTML/HTMLElement.idl>
```

Comment:
I don't think we need HTMLElement.idl here?

Reply:
true 👍🏼